i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

```diff
@@ -0,0 +1,150 @@
+- [API] Added a route to create a token.
+- [API] Added a route to delete a token.
+- [API] Added a route to list API tokens.
+- [ACL] Added action & menu access to manage the API tokens.
+- [API] The field "notify" has been added to the monitoring/services GET route.
+- [API] Removed is_activated field from host templates and service templates routes.
+- [API] Updated the service template creation route to handle relations with service groups.
+- [API] Updated the service template partial update route to handle relations with service groups.
+- [API] CLAPI now ignores empty lines and lines starting with the `#` character.
+- [API] Updated the service template create route to handle macros.
+- [API] Updated the service template partial update route to handle macros.
+- [API] Updated the service template partial update route to handle all properties.
+- [API] Updated the service template create route to handle all properties.
+- [API] Updated the service template partial update route to link a service template to service categories.
+- [API] Updated the host template create route to handle all properties.
+- [API] Added a route to handle templates of a host template (PATCH).
+- [API] Updated the service template partial update route to link a service template to host templates.
+- [API] Added a route to update a host categories of a host template (PATCH).
+- [API] Added a route to link a service template to host templates (PATCH).
+- [API] Updated the AddHostTemplate route to handle macros (POST)
+- [API] Added a route to update a host categories of a host template (POST).
+- [API] Added a route to update macros of a host template (PATCH).
+- [API] Added a route to handle templates of a host template (POST).
+- [API] Updated the host create API route to handle host groups.
+- [API] Added a route to list host templates.
+- [API] Added a route to update/find host severity.
+- [API] Added a route to get a host category.
+- [API] Added a route to update a host category.
+- [API] Added a route to update one host group (PUT).
+- [API] Added a route to retrieve one host group (PUT).
+
+</details>
+
+<details>
+<summary>Bug fixes</summary>
+- [API] Fixed an issue where values for contact additive inheritance was not updated with new endpoints.
+- [API] Fixed issues on duplicate display, encoding and add_inherited_contact in host and host template endpoints.
+- [API] Fixed access to list of groups of resources regarding ACL definition.
+- [CLAPI] Fixed a bug that duplicated the host template / service template relations.
+- [CLAPI] Removed the possibility to link a regular resource with a template.
+- [CLAPI] Added the missing parameters to handle new Broker logging options (thanks to a [community member's contribution](https://github.com/centreon/centreon/pull/885)). 
+- [CLAPI] Fixed the export of severities to be able to reimport them.
+- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
+</details>
+
```

Comment:
```suggestion
<details open>
  <summary>Breaking Change</summary>
- We removed the “persistent” option from the modal allowing to set acknowledgement because these one need to be persistant in case of Centreon Engine restart.
- [CLAPI] Macros are now displayed identically for host, host template and service template.
</details>
```